src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -44,6 +46,37 @@ public JsonSerializerOptions()
             Converters = new ConverterList(this);
         }
 
+        /// <summary>
+        /// Copies the options from a <see cref="JsonSerializerOptions"/> instance to a new instance.
+        /// </summary>
+        /// <param name="options">The <see cref="JsonSerializerOptions"/> instance to copy options from.</param>
+        public JsonSerializerOptions(JsonSerializerOptions options)
+        {
+            _dictionaryKeyPolicy = options._dictionaryKeyPolicy;
+            _jsonPropertyNamingPolicy = options._jsonPropertyNamingPolicy;
+            _readCommentHandling = options._readCommentHandling;
+            _referenceHandling = options._referenceHandling;
+            _encoder = options._encoder;
+
+            _defaultBufferSize = options._defaultBufferSize;
+            _maxDepth = options._maxDepth;
+            _allowTrailingCommas = options._allowTrailingCommas;
+            _ignoreNullValues = options._ignoreNullValues;
+            _ignoreReadOnlyProperties = options._ignoreReadOnlyProperties;
+            _propertyNameCaseInsensitive = options._propertyNameCaseInsensitive;
+            _writeIndented = options._writeIndented;
+
+            Converters = new ConverterList(this);
+            for (int i = 0; i < options.Converters.Count; i++)
+            {
+                Converters.Add(options.Converters[i]);
+            }
+
+            // _memberAccessorStrategy is not copied as that is platform specific.
```

Comment:
Can you help me understand this better?  Why does it being platform-specific mean we don't want to copy it?

Reply:
Removed. The comment didn't make sense as the conditions specified in the if directive below will apply to the new options as well:

```C#
internal MemberAccessor MemberAccessorStrategy
{
    get
    {
        if (_memberAccessorStrategy == null)
        {
#if NETFRAMEWORK || NETCOREAPP
            _memberAccessorStrategy = new ReflectionEmitMemberAccessor();
#else
            _memberAccessorStrategy = new ReflectionMemberAccessor();
#endif
        }

        return _memberAccessorStrategy;
    }
}
```